### PR TITLE
feat(docs): Update README.md and modify pre-commit hook

### PR DESCRIPTION
**Pull request by pamelaluquemck**
## Issue ticket number and link
https://trello.com/c/Tgj95B6d/296-azure-actualizar-el-readme

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My changes generate no new warnings or errors

## Pull request type

Please check the type of change your PR introduces:

- [ ] :sparkles: Feature Implementation
- [ ] :bug: Bugfix
- [ ] :art: Code Formatting (formatting, renaming)
- [ ] :recycle: Refactoring
-  [x] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :hammer: Automated testing
- [ ] :zap:  Dependency update
- [ ] :wrench: Add or update configuration files

## What is the current module?

- Pre-commit hook modify README.md 

## What is the new behavior?

- Pre-commit hook modify ./docs/README-tech.md and there is an update of README.md

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by pramirezma — CHANGES_REQUESTED** [review]
This doesn't follows terraform documentation practices, also, the readme steps are unnecessary when you have a pipeline installed. And there are heavily opinionated of your installation, the use of terraform and all services SHOULD be portable.
please don't upload init files with credentials.

File: README.md [hunk 1]
```diff
@@ -1,91 +1,106 @@
...
+      ```bash
+      docker buildx build --platform linux/amd64 -t image-mongo:v1 . -f Dockerfile
+      docker tag image-mongo:v1 azcapabilitiesacr.azurecr.io/image-mongo:v1
+      docker push azcapabilitiesacr.azurecr.io/image-mongo:v1
+      ```
+  - Deploy all the resources 
+  ```bash
+  terraform apply --var-file=dev.auto.tfvars
+  ```
+## Test the deployment
```
Comment (jose-sal-y-rosas) [inline README.md:90]: Please move all this section to another README file.

File: README.md [hunk 2]
```diff
@@ -1,91 +1,106 @@
...
+  -  Connect to Azure and connect the Azure Container Registry variables
+    ```bash
+      az login --service-principal --username $clientId --password $clientSecret --tenant $tenantId
+      ACR_LOGIN_SERVER=$(az acr show --name azcapabilitiesacr --query loginServer --output tsv)
+      az acr login --name $ACR_LOGIN_SERVER
+    ```
+  - Upload the images:
+    - [Frontend](https://github.com/CHUPITO-Org/FE-Chupito): 
+      ```bash
+      docker tag [frontend-image] azcapabilitiesacr.azurecr.io/ms-conference-ui
```
Comment (jose-sal-y-rosas) [inline README.md:71]: use the same format of the image to identify container registry. Avoid using real names.

File: README.md [hunk 3]
```diff
@@ -1,91 +1,106 @@
...
+      ```
+    - [Backend](https://github.com/CHUPITO-Org/BE-Chupito/): 
+      ```bash
+      docker tag [backend-image] azcapabilitiesacr.azurecr.io/ms-conference-bff
+      docker push azcapabilitiesacr.azurecr.io/ms-conference-bff
+      ```
+    - [Database](./database-image/Dockerfile): 
+    Locate in [./database-image](./database-image/Dockerfile) folder and execute:
+      ```bash
+      docker buildx build --platform linux/amd64 -t image-mongo:v1 . -f Dockerfile
```
Comment (jose-sal-y-rosas) [inline README.md:82]: What is the purpose of buildx here?

File: dev.auto.example.tfvars [hunk 4]
```diff
@@ -0,0 +1,7 @@
+client_id        = "Change this value"
```
Comment (jose-sal-y-rosas) [inline dev.auto.example.tfvars:1]: Rename this file according to the comment in the README file.

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review feedback

File: README.md (hunk 2 above)
Comment (pamelaluquemck) [reply README.md:71]: Ok

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

File: docs/ADR.md [hunk 5]
```diff
@@ -15,7 +25,7 @@
     * Public Ip
     * Storage Account
 
-## Infrastructure Cost
+### Infrastructure Cost
 
 Costo por mes durante 24 horas:
```
Comment (pramirezma) [inline docs/ADR.md:30]: id this supposed to be in Spanish?

File: testing.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+## Test the deployment
+Navigate in your browser:http://az-capabilities-dns.eastus2.cloudapp.azure.com/
```
Comment (pramirezma) [inline testing.md:2]: this should depend of the cloud app name, if you change the variable you change this hostname

File: README.md [hunk 7]
```diff
@@ -1,91 +1,62 @@
...
+| :-------------------------| :-------------------------------------------------------- |
+| **client_id**             | App registration credential: Client ID                    |
+| **client_secret**         | App registration credential: Client Secret                |
+| **subscription_id**       | App registration credential: Subscription ID              |
+| **tenant_id**             | App registration credential: Tenant ID                    |
+
+## Usage
+
+### Local deployment
+
```
Comment (pramirezma) [inline README.md:48]: do we need "local deployment"

**Review by pramirezma — CHANGES_REQUESTED** [review]

File: docs/ADR.md (hunk 5 above)
Comment (pamelaluquemck) [reply docs/ADR.md]: No, I'll translate to english

File: testing.md (hunk 6 above)
Comment (pamelaluquemck) [reply testing.md]: I'll put a general name instead

File: README.md (hunk 7 above)
Comment (pamelaluquemck) [reply README.md]: Could be an option. But as we are using terraform cloud now I will remove from the readme

File: docs/ADR.md (hunk 5 above)
Comment (pramirezma) [reply docs/ADR.md]: Still saying "Costo por mes durante 24 horas:"

Comment (pramirezma) [reply docs/ADR.md]: now is displaying correctly

**Pull request merged**